### PR TITLE
Add missing links

### PR DESCRIPTION
**Pull request by roundminer**
#6  fixed
I have fixed markdown to contain links as in original article.

I have provided my ETH address on openbounty.status.im, same username as here: roundminer.

Bounty and donation ETH address:
0xA9b7226D8a0Daf91F614c5E7511C3E10d6e0B10c

### CODE REVIEW

**Comment by roundminer** [comment]
Also, added fixed links for second article. So, I have fixed issues: #6 #7

**Comment by Bradymck** [comment]
All the different projects that will be involved have a similar blog footer with links to more info. Though for this publication a "canned" footer with the website may be good to add at the very least. Thoughts?

File: docs/articles/editorial/the_district0x_in_a_nutshell.md [hunk 1]
```diff
@@ -7,31 +7,36 @@ _This article was originally posted in the [Aragon Blog](https://blog.aragon.one
 
 ![](images/district0x_nutshell01.jpeg)
 
-The district0x project is building a general purpose framework of smart contracts and front end libraries for the purpose of deploying, operating, and governing communal marketplaces.The network is powered by Ethereum smart contracts, ‘Aragon’ governance and the distributed file system ‘IPFS’.
+The district0x project is building a general purpose framework of smart contracts and front end libraries for the purpose of deploying, operating, and governing communal marketplaces.The network is powered by [Ethereum](https://www.ethereum.org/) smart contracts, ‘[Aragon](https://aragon.one/)’ governance and the distributed file system ‘[IPFS](https://ipfs.io/)’.
 
 The community has asked for a simple breakdown of the district0x network and the following post is based on this community feedback. Let’s dive in…
 
-## Our Vision
+### Our Vision
```
Comment (Smokyish) [inline docs/articles/editorial/the_district0x_in_a_nutshell.md]: Why was this changed to `h3`?

File: docs/articles/opinion/thoughts_on_governance_network_effects.md [hunk 2]
```diff
@@ -6,84 +6,96 @@ _This article was originally posted in the [Aragon Blog](https://blog.aragon.one
...
+If the market is commoditized and your competitors have at best an equivalent product to yours, issuing a token and requiring it to be used in order to access your product works (sort of like an [API key](https://en.wikipedia.org/wiki/Application_programming_interface_key)). However, by forcing the use of a token when your application doesn’t require one you create a worse user experience because the user’s effective cost is unpredictable. Now if a competitor offers the same product without the restriction, it isn’t equivalent, it is objectively better. Therefore, differentiation in this way may work to some degree, but is not particularly **defensible**.
+
+#### Defensible tokenization
```
Comment (Smokyish) [inline docs/articles/opinion/thoughts_on_governance_network_effects.md]: These header changes are breaking changes because of how the css is structured and should not be changed, request was for editing links **only**

File: docs/articles/opinion/thoughts_on_governance_network_effects.md [hunk 3]
```diff
@@ -6,84 +6,96 @@ _This article was originally posted in the [Aragon Blog](https://blog.aragon.one
...
 
-Token holders are the only party whose interest we can programmatically quantify. Other relevant parties exist, including users, developers, miners, and validators, but their interests in the network are not programatically quantifiable. By entrusting decisions which impact the network to token holders, they become the keepers of the network effect. The service they are providing is formal representation of the informal interests of all network participants.
+Token holders are the only party whose interest we can programmatically quantify. Other relevant parties exist, including users, developers, miners, and validators, but their interests in the network are not programatically quantifiable. By entrusting decisions which impact the network to token holders, they become the [keepers](https://medium.com/@rzurrer/keepers-workers-that-maintain-blockchain-networks-a40182615b66) of the network effect. The service they are providing is formal representation of the informal interests of all network participants.
 
-![](images/thoughts_on_governance02.png)
+<figure name="f7e3" id="f7e3" class="graf graf--figure graf--layoutOutsetLeft graf-after--p">
```
Comment (Smokyish) [inline docs/articles/opinion/thoughts_on_governance_network_effects.md]: This should not be changed!

File: docs/articles/opinion/thoughts_on_governance_network_effects.md [hunk 4]
```diff
@@ -6,84 +6,96 @@ _This article was originally posted in the [Aragon Blog](https://blog.aragon.one
...
 
-The model is defensible because **[A]** fees can be justified for the service token holders provide (making decisions that ensure the long term sustainability and growth of the network). If token holders do a bad job, or charge excessive fees, the network will fail, but if they do a good job they will be able to adapt to changing market conditions and be more resilient to the effects of commoditization. **[B]** Properties like stability are not critical to participating in governance. **[C]** Using a unique token is required because it quantifies a users influence in order to align incentivizes to ensure the service provided is consistent.
+<div class="aspectRatioPlaceholder">![](https://cdn-images-1.medium.com/max/600/1*ZLU-gZhzyik3WkM_f8-DVA.png)</div>
+
+<figcaption class="imageCaption">Governance as a Service allows for differentation in markets were the hardware, software, and data have become commoditized.</figcaption>
```
Comment (Smokyish) [inline docs/articles/opinion/thoughts_on_governance_network_effects.md]: Another very unnecessary change.

**Comment by Smokyish** [comment]
@roundminer Because this was merged before my review, please see all the comments and the review above and submit a new Pull Request which also follows our [Pull Request template](https://monthly.aragon.one/guides/new_pull_request_template/).

The changes introduced here [have been reverted and](https://github.com/aragon/aragon-monthly/pull/10) the Pull Request rejected as is.

**Comment by roundminer** [comment]
Fixed in new PR #11
